From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Thu, 1 Mar 2018 17:27:50 +0100
Subject: staging: fsl-mc: dpio: remove foolish -Werror Makefile addtion

Git-commit: 0540bccdb5afe856958ed57b20e8e9a75ff9c3b7
Patch-mainline: v4.17-rc1
References: fate#326530,fate#326531,fate#326535,fate#326538,fate#326539

With the zillion different compilers out there, never use -Werror,
otherwise your code will end up breaking the build for odd reasons.

Like now, if this driver is enabled, it breaks the build due to a
function that could be marked static.  So it's obvious no one is even
paying attention to this driver :(

Cc: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Mian Yousaf Kaukab <yousaf.kaukab@suse.com>
---
 drivers/staging/fsl-mc/bus/dpio/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/staging/fsl-mc/bus/dpio/Makefile b/drivers/staging/fsl-mc/bus/dpio/Makefile
index 53ba84d7b884..b9ff24c76582 100644
--- a/drivers/staging/fsl-mc/bus/dpio/Makefile
+++ b/drivers/staging/fsl-mc/bus/dpio/Makefile
@@ -3,8 +3,6 @@
 # QorIQ DPAA2 DPIO driver
 #
 
-subdir-ccflags-y := -Werror
-
 obj-$(CONFIG_FSL_MC_DPIO) += fsl-mc-dpio.o
 
 fsl-mc-dpio-objs := dpio.o qbman-portal.o dpio-service.o dpio-driver.o
-- 
2.11.0

